nexus_constructor/json/load_from_json.py

```diff
@@ -91,6 +90,7 @@ def _retrieve_children_list(json_dict: Dict) -> List:
         return entry[CommonKeys.CHILDREN]
     except (KeyError, IndexError, TypeError):
         return []
+    return []
```

Comment:
Didn't get why this was added?

Reply:
@ebadkamil I second your comment. Could you push it to the branch? Also, maybe instead of pass, put print(error) so the user might see if it is KeyError, IndexError or TypeError.

Reply:
Not sure if we do need to print(error) here. May be of no use for users.

Reply:
Yeah, it is a shame it is a function outside of the class, because I believe we could have appended it to the warnings instead if not.